scripts/suite_objects.py

```diff
@@ -1360,18 +1360,32 @@ def add_var_debug_check(self, var):
                 if not ':' in dim:
                     dim_var = self.find_variable(standard_name=dim)
                     if not dim_var:
-                        raise Exception(f"No dimension with standard name '{dim}'")
-                    self.update_group_call_list_variable(dim_var)
+                        # DJS2025: To allow for numerical dimensions in metadata.
```

Comment:
I am not a fan of this type of comment. Could you remove all the `DJS2025:` bits now that this feature works?
```suggestion
                        # To allow for numerical dimensions in metadata.
```